backend/apps/ml_triage/services/github_collector.py

```
@@ -0,0 +1,139 @@
+        issues = []
+        page = 1
+        
+        while len(issues) < limit:
+            url = f"{self.base_url}/repos/{repo}/issues"
+            params = {
+                'state': state,
+                'page': page,
+                'per_page': 100
+            }
+            
+            response = requests.get(url, headers=self.headers, params=params)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)